java/query/src/main/java/sleeper/query/model/output/S3ResultsOutput.java

```diff
@@ -70,25 +77,24 @@ public S3ResultsOutput(InstanceProperties instanceProperties, Schema schema, Map
         String defaultRowGroupSize = instanceProperties.get(DEFAULT_RESULTS_ROW_GROUP_SIZE);
         String defaultPageSize = instanceProperties.get(DEFAULT_RESULTS_PAGE_SIZE);
 
-        this.compressionCodec = config.getOrDefault(COMPRESSION_CODEC, "zstd");
         this.fileSystem = instanceProperties.get(FILE_SYSTEM);
         this.rowGroupSize = Long.parseLong(config.getOrDefault(ROW_GROUP_SIZE, defaultRowGroupSize));
         this.pageSize = Integer.parseInt(config.getOrDefault(PAGE_SIZE, defaultPageSize));
+        this.compressionCodec = tableProperties.get(sleeper.configuration.properties.table.TableProperty.COMPRESSION_CODEC);
```

Comment:
This could be TableProperty.COMPRESSION_CODEC, rather than the fully qualified reference. Another option would be to rename the COMPRESSION_CODEC static final field in this class.

Reply:
Done.